### ISSUE TITLE
Android app stops playing when another player opened

### ISSUE THREAD

**Issue by BahodurSaidov** [florent37/Flutter-AssetsAudioPlayer#79]
Hi,
I just build apk --release the existing example and tested on android 9. The music player functionalities work well. When another music player opened in parallel (and a music is played) the flutter app audio pauses and doesn't play audio again, even if different music selected...
But the notification sound (bell icon) plays without problem. 
Also, i noticed that when a system notification sounds play (like sms) it just volumes down for a while and continues playing without break.

Expectations:
Music should continue playing even if another player is playing sound.

Reproducing steps:
cd example
rename android folder to android_old
flutter create .
flutter build apk --release -v
install built flutter app on android device, open it, play Rock music, go to another music player and play some music file, then reopen flutter app from background, press play button, press bell button.

**Comment by florent37** [comment]
It's a regression due to a recent bugfix,

what's your assets_audio_player version ?

**Comment by florent37** [comment]
```
Also, i noticed that when a system notification sounds play (like sms) it just volumes down for a while and continues playing without break
```

yes It's because of the audio_focus implemented
you have on android 2 main focus loose mode : 

AUDIOFOCUS_LOSS -> need to pause the player
AUDIOFOCUS_LOSS_TRANSIENT_CAN_DUCK -> need to reduce the volume

https://developer.android.com/guide/topics/media-apps/audio-focus

it's the default behavior, but I can if you want add an option on the player to make it "pause" on AUDIOFOCUS_LOSS_TRANSIENT_CAN_DUCK

**Comment by BahodurSaidov** [comment]
> It's a regression due to a recent bugfix,
> 
> what's your assets_audio_player version ?

my used version is 1.0.1

**Comment by BahodurSaidov** [comment]
> it's the default behavior, but I can if you want add an option on the player to make it "pause" on AUDIOFOCUS_LOSS_TRANSIENT_CAN_DUCK

yes it looks good, it is normal implementation. I just mentioned this so to be sure that only music player sound interrupts the flutter app not the notification sounds...

**Comment by florent37** [comment]
can you try the last version :   `assets_audio_player: ^1.4.6` ?

**Comment by BahodurSaidov** [comment]
> can you try the last version : `assets_audio_player: ^1.4.6` ?

how to update? should i re-clone whole app and android project files ?

**Comment by florent37** [comment]
how did you get the sample ? 

if it's by git, use `git pull`

else, re-clone the project

**Comment by florent37** [comment]
I'm on android 10 btw, I'll try tomorrow with android 9

**Comment by BahodurSaidov** [comment]
hi, @florent37 
great, nice notification buttons, and works well even if other players are on the way. I think you can close this issue.

**Comment by BahodurSaidov** [comment]
But i tested same app on android 4.2.2 and it doesnt play even bell button. And the app breaks unexpectedly. But i suspect that this issue is related to flutter tools. Any idea ? (may be you should open another issue..)

**Comment by florent37** [comment]
Thanks :)

**Comment by florent37** [comment]
Yes you can open another issue for this. Btw I think I'll set an Android minSdk to 21 (5.0)

**Comment by florent37** [comment]
I close it because you opened a new issue for 4.2 Android
